src/tests/tracing/eventpipe/common/Reverse.cs

```diff
@@ -101,8 +101,7 @@ public ReverseServer(string serverAddress, int bufferSize = 16 * 1024)
                 socket.SendBufferSize = Math.Max(bufferSize, 1024);
                 socket.ReceiveBufferSize = Math.Max(bufferSize, 128);
                 socket.Bind(remoteEP);
-                socket.Listen(255);
-                socket.LingerState.Enabled = false;
+                socket.Listen(0);
```

Comment:
Based on the issue description I wouldn't expect this change to be needed. Do things still work with this part reverted?

Reply:
Ah, these didn't need to make it in.  It turns out that the `SO_LINGER` socket option only applies to protocol based sockets, e.g., TCP sockets, so it shouldn't actually have an effect here.  The other option shouldn't change anything so I'll revert it back.